### ISSUE TITLE
Valid html is reported as one that has "end-tag" warning

### ISSUE THREAD

**Issue by ZuBB** [markuplint/markuplint#1349]
- Markuplint version: `3.15.0`
- Parser lang: `nunjucks`
- Node.js version: `v18.18.2`
- OS: macOS

## Describe the bug
Validation of  Django template (basically html + nunjucks) ends with `end-tag` warning

## Code Example

<details>
<summary>Click me</summary>

```html
{% spaceless %}
{% load i18n %}

<footer>
    <div class="border-top"></div>

    <div class="root-container main-footer-block d-flex flex-wrap align-items-center mx-auto">
        <div class="footer-logo d-none d-md-block h-100">
            {% include "includes/common-widgets/logo.html" with monochrome=True %}
        </div>

        <div id="contacts" class="contacts-block h-100">
            {% include "includes/common-widgets/address.html" %}
        </div>

        {% with smm_class="btn btn-sm btn-secondary btn-iconified btn-right-arrow-icon w-100" %}
        <div>
            {% with li_href="https://www.linkedin.com/company/..." %}
            <a href="{{ li_href }}" class="{{ smm_class }} btn-smm-li">LinkedIn</a>
            {% endwith %}
        </div>

        <div>
            {% with ig_href="https://www.instagram.com/..." %}
            <a href="{{ ig_href }}" class="{{ smm_class }} btn-smm-ig">Instagram</a>
            {% endwith %}
        </div>

        <div>
            {% with fb_href="https://www.facebook.com/..." %}
            <a href="{{ fb_href }}" class="{{ smm_class }} btn-smm-fb">Facebook</a>
            {% endwith %}
        </div>
        {% endwith %}

        <div class="scroll-to-top d-none d-md-block">
            {% include "includes/common-widgets/button/scroll-to-top.html" %}
        </div>
    </div>

    <div class="border-top"></div>

    <div class="root-container mx-auto">
        {% include "includes/common-widgets/copyright-policies.html" with
            scroll_button=True
        %}
    </div>
</footer>
{% endspaceless %}
```
</details>

## Steps To Reproduce

1. Just lint the code from the example

## The current behavior

See screen
<img width="1728" alt="image" src="https://github.com/markuplint/markuplint/assets/597016/2e342d35-44ed-4d66-b28a-12617327016f">

## The expected behavior

No warning should be reported.

**Comment by YusukeHirao** [comment]
@ZuBB 

Thank you for reporting this issue. We are aware that Markuplint has limitations in supporting certain complex attribute syntaxes. Ex: `class="{{ smm_class }} btn-smm-fb"`
See: https://markuplint.dev/docs/guides/besides-html#-unavailable-code
Ref Issue: https://github.com/markuplint/markuplint/issues/240

We are keen to address this issue, but given our current development resources and other priorities, it has a somewhat lower priority. However, we understand the importance of this problem and aim to improve it in the future.

Providing a specific timeline for addressing this issue is difficult, but we will update the issue thread as progress is made. Contributions or suggestions from the community regarding this problem are also welcome. We appreciate your understanding and cooperation.

**Comment by ZuBB** [comment]
@YusukeHirao,

thanks for pointing this out. I probably missed that while I was reading the docs. I am sorry for that.
